### PR TITLE
Fix #20496: Relax requirements for compact inverted coasters

### LINKED ISSUE #20496: Compact Inverted Coaster no longer has relaxed stat requirements if it has inversions
### Operating System

Windows 10

### OpenRCT2 build

OpenRCT v0.4.5

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This is a development issue

### Describe the issue

The Compact Inverted is supposed to have its requirement for the drop height nullified if it has at least one inversion, but as of v0.4.5 that's no longer the case.

I believe this line should say true instead of false: https://github.com/OpenRCT2/OpenRCT2/blob/d40102aa9ded9d8ef6d5e434d9dcde959ced44dc/src/openrct2/ride/coaster/meta/CompactInvertedCoaster.h#L62C11-L62C11

### Steps to reproduce

Build the attached ride design both in version 0.4.4 and 0.4.5 and see how the stats are halved in 0.4.5. 

### Attachments

[Inverted Vertical Shuttle cheap.zip](https://github.com/OpenRCT2/OpenRCT2/files/11880582/Inverted.Vertical.Shuttle.cheap.zip)

### PR DESCRIPTION
**Pull request by aramk-hub**
Fixed #20496 where compact inverted coasters did not get the relaxed requirements when there were inversions since the boolean for it was originally set to false.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Are you able to work out when it was changed?

**Comment by aramk-hub** [comment]
Seems like it was introduced with fe219281ec6b66e3a47c67125c939bc1f01d1274 as false, and wasn't in the file before this

**Review by ZehMatt — APPROVED** [review]
This needs a changelog entry.

Note to @OpenRCT2/developers, Given that all tests pass we probably have no park that covers this case currently, we should update the EverythingPark to include this.

**Comment by Broxzier** [comment]
> Given that all tests pass we probably have no park that covers this case currently, we should update the EverythingPark to include this.

The original issue has a track design already provided, so it should be as simple as just placing this. Do replays recalculate all ride stats, though? Otherwise it's pointless since it's just checking cached data.

**Comment by ZehMatt** [comment]
> > Given that all tests pass we probably have no park that covers this case currently, we should update the EverythingPark to include this.
> 
> The original issue has a track design already provided, so it should be as simple as just placing this. Do replays recalculate all ride stats, though? Otherwise it's pointless since it's just checking cached data.

We have separate tests for ride ratings

**Comment by ZehMatt** [comment]
Also forgot that this needs a network version bump.

**Comment by ZehMatt** [comment]
@aramk-hub could you raise the network version? Then it would be good to go.

**Comment by SpartanFrederic104** [comment]
Please keep this open if it's still being worked on

**Review by ZehMatt — APPROVED** [review]

File: distribution/changelog.txt [hunk 1]
```diff
@@ -33,6 +33,7 @@
 - Fix: [#20429] Error window tooltip not closing after 8 seconds.
 - Fix: [#20456] Downward large half loops on flying coasters (fly-to-lie) are now correctly named.
 - Fix: [#20484] Console caret not properly updated when using command history.
+- Fix: [#20496] Relax ride rating requirements for compact inverted coasters.
```
Comment (Broxzier) [inline distribution/changelog.txt:36]: This should say what was broken, not what has changed.

```suggestion
- Fix: [#20496] Ride rating requirements for compact inverted coasters is no longer relaxed.
```

**Review by Broxzier — CHANGES_REQUESTED** [review]

**Review by Broxzier — APPROVED** [review]

**Pull request merged**